schemas/lst/SCHEMA.md

```diff
@@ -0,0 +1,19 @@
+# Liquid Staking Tokens (LSTs) Schema
+
+This is the OpenBlock Labs standard schema for liquid staking tokens.
+
+## Schema
+
+### Position Snapshot
+
+**Description:** User level snapshot of holders in this protocol.
+
+| Property       | Description                                                      | Type   |
+|----------------|------------------------------------------------------------------|--------|
+| timestamp      | The timestamp of the snapshot.                                   | number |
+| chain_id       | Standard chain ID.                                               | number |
+| token_address  | The address of the LST in the protocol.                          | string |
```

Comment:
is this just contract_address of the erc20 LST ?

Reply:
Yeah